### PR TITLE
Fixed ADN once again.

### LINKED ISSUE #3720: [Presence | Bug] Anime Digital Network
**Describe the Bug**
The status is displaying "Browsing...", but starting an episode doesn't change the status.

**Steps to Reproduce**
1. Open https://animedigitalnetwork.fr/
2. Start an episode

**Expected Behavior**
The status should be updated with anime title, episode...

**Screenshots**
Screenshot_220

**Environment**
- OS: Windows 10
- Browser: Chrome
- Version: 90.0.4430.212 (64-bit)

### PR DESCRIPTION
**Pull request by DarkVillager**
Resolves #3720
image

### CODE REVIEW

**Review by ririxidev — CHANGES_REQUESTED** [review]
move objects with PresenceData to one and for example at function start.

**Comment by DarkVillager** [comment]
> move objects with PresenceData to one and for example at function start.

?

**Comment by ririxidev** [comment]
create presenceData object after `presence.on` and use it instead of creating objects everytime

File: websites/A/AnimeDigitalNetwork/presence.ts [hunk 1]
```diff
@@ -8,46 +8,40 @@ strings = presence.getStrings({
 
 presence.on("UpdateData", async () => {
 const video: HTMLVideoElement = document.querySelector("video.vjs-tech");
+const presenceData: PresenceData = {
```
Comment (ririxidev) [inline websites/A/AnimeDigitalNetwork/presence.ts:11]: merge

File: websites/A/AnimeDigitalNetwork/presence.ts [hunk 2]
```diff
@@ -8,46 +8,40 @@ strings = presence.getStrings({
...
-      state: title.textContent,
-      largeImageKey: "logo"
-    };
-  presence.setActivity(data);
+  const title = document.querySelector("#root > div > div > div.sc-pbWVv.hTvDIL > div > div:nth-child(1) > div.sc-kbKFCX.sc-kgMcbC.kecmmo > div:nth-child(1) > div > div > h1 > a");
+      presenceData.details = "Looking at";
+      presenceData.state = title.textContent;
+} else {
+  presenceData.details = "Browsing...";
+}
```
Comment (ririxidev) [inline websites/A/AnimeDigitalNetwork/presence.ts:40]:
```suggestion
} else presenceData.details = "Browsing..."
```

**Review by ririxidev — CHANGES_REQUESTED** [review]

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**